.github/workflows/nightly-failed.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Nightly build failed
+assignees: iNikem
```

Comment:
Is it only possible to assign one person to assignee? If not, maybe can assign the team if it's possible, or otherwise the manually maintained list of the maintainers if that's possible. Otherwise... thanks! ;)

Reply:
Several assignees can be defined. Don't know about a team. I am OK for now with being single assignee or I can add you and @trask, if you want :)

Reply:
I can't speak for @trask but go ahead and add me :)

Reply:
does it need an assignee? or can we have no assignee, and then whichever one of us picks it up can assign ourselves so we don't duplicate effort fixing

Reply:
I propose to leave it as it is, assigning to me (it will help me react faster), but you are more than welcomed to steal these issue from me if you reach them earlier :)

Reply:
that works for me, what time does the nightly build run, and should we optimize that timing for you?

Reply:
03:24 UTC. That is usual nightly time for me :) So both you and Anuraag can grab it immediately, or I will wake up and deal with it.

Reply:
> or I will wake up and deal with it

please don't 😄

and feel free to move the timing of the nightly build any time you want

Reply:
I mean "wake up in my usual time in the morning and will deal with it in a due course during my work day" :)